### ISSUE TITLE
 Publish ELK Random Graph Generator to Maven Repository 

### ISSUE THREAD

**Issue by ningpp** [eclipse/elk#907]
The ELK is very useful for me, but i can't find any grandom jar in Maven Central Repository.

**Comment by soerendomroes** [comment]
Yes, this is the case since it only holds developer plugins for Eclipse to generate random graphs. What do you need it for?

**Comment by ningpp** [comment]
gen random graph to test graph layout  @soerendomroes

**Comment by soerendomroes** [comment]
You could get the artifacts from the update site https://download.eclipse.org/elk/updates/releases/0.8.1/.
Since it is Eclipse dependent it is not currently not released on maven central.

**Comment by ningpp** [comment]
thank you @soerendomroes

**Comment by soerendomroes** [comment]
Do you know how to write the generation files? If not, please have a look at this guide https://www.eclipse.org/elk/documentation/algorithmdevelopers/randomgraphs.html

**Comment by ningpp** [comment]
> Do you know how to write the generation files? If not, please have a look at this guide https://www.eclipse.org/elk/documentation/algorithmdevelopers/randomgraphs.html

thank you
